src/app/services/constants.ts

```diff
@@ -46,9 +46,9 @@ export const GOOGLE_ANALYTICS_4_MEASUREMENT_ID = envSpecific("G-H95WP5C8DR", "G-
 
 export const SOCIAL_LINKS = {
   youtube: { name: "YouTube", href: "https://www.youtube.com/@isaaccomputerscience" },
-  twitter: { name: "Twitter", href: "https://twitter.com/isaaccompsci" },
   facebook: { name: "Facebook", href: "https://www.facebook.com/IsaacComputerScience" },
   instagram: { name: "Instagram", href: "https://www.instagram.com/isaaccompsci" },
+  twitter: { name: "Twitter", href: "https://twitter.com/isaaccompsci" },
```

Comment:
is there chance to change this to be `X` now that twitter no longer exists by that name?